Client/Frontend/Browser/TabType.swift

```diff
@@ -0,0 +1,30 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+enum TabType: Int, CustomDebugStringConvertible {
+
+    case regular
+    case `private`
+
+    var isPrivate: Bool {
+        switch self {
```

Comment:
I use `switch` statements so that a compiler error is thrown when a new 'case' is added, making sure that we think about the implementation for each case